plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -49,7 +49,7 @@ class ServiceActionDisabledModal extends React.Component {
 
     // For everything SDK, not other packages
     if (isSDKService(service)) {
-      return `dcos ${packageName} --name=${serviceID} update --options=${packageName}-options.json`;
+      return `dcos ${packageName} --name=${serviceID} update start --options=${packageName}-options.json`;
```

Comment:
Let's just make `--options=options.json` and remove the packageName from it. I think that's easier to follow given the copy above the command.